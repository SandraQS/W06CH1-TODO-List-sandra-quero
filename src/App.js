import "./App.css";
import Form from "./components/Form/Form";
import ListToDo from "./components/ListToDo/ListToDo";

function App() {
  return (
    <>
      <Form />
      <h1>To Do List</h1>
      <ListToDo />
    </>
  );
}

export default App;
