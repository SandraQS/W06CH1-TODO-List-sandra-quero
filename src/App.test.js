import { Provider } from "react-redux";
import App from "./App";
import configureStore from "./redux/store";
import ReactTestRenderer from "react-test-renderer";

describe("Given a App component", () => {
  describe("When it's rendered", () => {
    test("Then it should show match the last snapshot", () => {
      const store = configureStore();

      const app = ReactTestRenderer.create(
        <Provider store={store}>
          <App />
        </Provider>
      );

      expect(app.toJSON()).toMatchSnapshot();
    });
  });
});
