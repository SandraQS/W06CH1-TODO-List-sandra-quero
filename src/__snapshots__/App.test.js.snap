// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a App component When it's rendered Then it should show match the last snapshot 1`] = `
Array [
  <form
    onSubmit={[Function]}
  >
    <label
      className="form-label"
      htmlFor="exampleFormControlInput1"
    >
      Task
    </label>
    <input
      autoComplete="off"
      className="form-control"
      id="name"
      onChange={[Function]}
      placeholder="Task"
      type="text"
      value=""
    />
    <input
      className="btn btn-dark mb-3"
      type="submit"
      value="Create"
    />
  </form>,
  <h1>
    To Do List
  </h1>,
  <div
    className="form-check"
  />,
]
`;
